src/services/services.ts

```diff
@@ -427,6 +443,14 @@ namespace ts {
 
             return this.documentationComment;
         }
+
+        getJsDocTags(): JSDocTagInfo[] {
+            if (this.jsDocTags === undefined) {
+                this.jsDocTags = this.declaration ? JsDoc.getJsDocTagsFromDeclarations([this.declaration]) : [];
+            }

```

Comment:
How about using `undefined` for storing un-computed state and `null` for computed and empty?